src/status_im/browser/core.cljs

```diff
@@ -200,6 +200,12 @@
     (universal-links/handle-url cofx link)
     {:browser/show-browser-selection link}))
 
+(fx/defn browser-selection-cancel
```

Comment:
you can use `{:events [:browser.ui/browser-selection-cancel]}` in that case you don't need to register handler in events ns

Reply:
would that be preferable?

Reply:
We are moving to this new method so yes it would be preferable

Reply:
I don't completely follow, but is this what you mean? In `list_selection.cljs` the `browse` fn will be define as such:

```

(defn browse [link]
  (show {:title       (i18n/label :t/browsing-title)
         :options     [{:label  (i18n/label :t/browsing-open-in-status)
                        :action #(re-frame/dispatch [:browser.ui/open-url link])}
                       {:label  (i18n/label (platform-web-browser))
                        :action #(.openURL (react/linking) (http/normalize-url link))}]
         :cancel-text (i18n/label :t/browsing-cancel)
         :on-cancel   #(re-frame/dispatch {:events [:browser.ui/browser-selection-cancel]})}))
```
And this:
```
(handlers/register-handler-fx
 :browser.ui/browser-selection-cancel
 (fn [cofx [_ _]]
   (browser/browser-selection-cancel cofx)))
```
is no longer needed in the events ns?